public/app/features/explore/Table.tsx

```diff
@@ -28,7 +29,11 @@ export default class Table extends PureComponent<TableProps> {
           if (link.className === 'link') {
             const columnKey = column.Header;
             const rowValue = rowInfo.row[columnKey];
-            this.props.onClickCell(columnKey, rowValue);
+            const attributes = Array.from(link.attributes).reduce((attributes, attribute) => {
```

Comment:
I'm not a huge fan of data attributes, but ReactTable is already doing something weird here anyway by channelling all cell events through here, so I think it's actually the least painful way to get input to a handler.